packages/bot-web-ui/src/components/bot-notification/bot-notification.tsx

```diff
@@ -28,19 +36,19 @@ const BotSnackbar = ({ message, timeout = 6000, is_open, onClick, handleClose, t
             onMouseLeave={() => {
                 setNotificationTimer(timeout);
             }}
-            className={className ?? 'bot-snackbar'}
-            data-testid='bot-snackbar-notification-container'
+            className={className ?? 'bot-notification'}
+            data-testid='bot-notification-container'
```

Comment:
@farabi-deriv Bhai I know it's not your change but since we are touching this file can we put the `dt` prefix for the data-testid as we are following this convention?

Reply:
Sure will update it according to the convention and also on other files that are missing this. @shafin-deriv